run/axcrypt2john.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/python
 
-# AxCrypt 1.x encrypted file parser for JtR.
+# AxCrypt 1.x and 2.x encrypted file parser for JtR.
+#
 # Written in 2016 by Fist0urs <eddy.maaalou at gmail.com>.
 #
 # This software is Copyright (c) 2016, Fist0urs <eddy.maaalou at gmail.com>,
```

Comment:
You should probably add your own copyright/license blurb as well here

Reply:
Can't fool the bots though
```
opencl_axcrypt_fmt_plug.c:287:3: error: ‘axcrypt_tests’ undeclared here (not in a function)
   axcrypt_tests
   ^
make[1]: *** [opencl_axcrypt_fmt_plug.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [default] Error 2
```
Just a super quick guess is you forgot to source the variable header

Reply:
Yep, I wrote this on a computer without a working OpenCL setup. Fixing now.

Reply:
License stuff should be fixed too now.